<!doctype html><html lang=en><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=description content="Webpage of the Resendis Lab"><meta name=viewport content="width=device-width,initial-scale=1,minimum-scale=1"><meta name=generator content="Hugo 0.79.0"><title>Webpage of the Resendis Lab</title><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Lato:300,400,500,700" type=text/css><link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel=stylesheet><link rel=stylesheet href=https://storage.googleapis.com/code.getmdl.io/1.3.0/material.indigo-green.min.css><link rel=stylesheet href=../css/styles.css><link rel=icon type=image/png href=../media/favicon.png></head><body><div class="mdl-layout mdl-js-layout mdl-layout--fixed-drawer mdl-layout--fixed-header"><header class="mdl-layout__header mdl-layout--small-screen-only"></header><div class=mdl-layout__drawer><a class=mdl-layout-title href=../><img src=../media/logo.png width=80% align=middle style=margin-top:2em></a><nav class=mdl-navigation><a class=mdl-navigation__link href=../>Home</a>
<a class=mdl-navigation__link href=../about>About</a>
<a class=mdl-navigation__link href=../members>Members</a>
<a class=mdl-navigation__link href=../positions>Work with us</a>
<a class=mdl-navigation__link href=../pubs>Publications</a>
<a class=mdl-navigation__link href=../projects>Projects</a>
<a class=mdl-navigation__link href=../software>Software</a>
<a class=mdl-navigation__link href=../events>Events</a>
<a class=mdl-navigation__link href=../about/#directions>Contact</a></nav></div><main class="mdl-layout__content mdl-color--grey-100"><div class="page-content mdl-grid"><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet" style=margin-bottom:2em><div class="card mdl-card mdl-shadow--4dp"><div class="mdl-card__title mdl-color--indigo-900" style=height:auto><h1 class=mdl-card__title-text>Software</h1></div></div></div><div class="mdl-cell mdl-cell--12-col-desktop mdl-cell--8-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><img src=https://github.com/resendislab/micom/raw/master/micom.png style=height:4em;width:auto><h2 class=mdl-color-text--teal-500>Christian Diener and Osbaldo Resendis Antonio</h2><br><p>micom is a Python package for metabolic modeling of microbial communities
developed in the Human Systems Biology Group of Prof. Osbaldo Resendis Antonio
at the National Institute of Genomic Medicine Mexico.</p><p>micom allows you to construct a community model from a list on input COBRA
models and manages exchange fluxes between individuals and individuals with the
environment. It explicitly accounts for different abundances of individuals in
the community and can thus incorporate data from 16S rRNA sequencing
experiments. It allows optimization with a variety of algorithms modeling the
trade-off between egoistic growth rate maximization and cooperative objectives.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://github.com/resendislab/micom><i class=material-icons>code</i>&nbsp;Source</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://resendislab.github.io/micom><i class=material-icons>link</i>&nbsp;Documentation</a></div></div></div><div class="mdl-cell mdl-cell--12-col-desktop mdl-cell--8-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Our microbiome pipeline</h1><h2 class=mdl-color-text--teal-500>Christian Diener and Osbaldo Resendis Antonio</h2><br><p>This repository contains the standardized analysis pipeline for 16S and metagenome
data. It serves as a testing ground for what will be required to analyze around
500 samples.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://github.com/resendislab/microbiome><i class=material-icons>code</i>&nbsp;Source</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://resendislab.github.io/microbiome><i class=material-icons>link</i>&nbsp;Documentation</a></div></div></div><div class="mdl-cell mdl-cell--12-col-desktop mdl-cell--8-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>CORDA for Python</h1><h2 class=mdl-color-text--teal-500>Christian Diener and Osbaldo Resendis Antonio</h2><br><p>This is a Python implementation based on the papers of Schultz et. al. with
some added optimizations. It is based on the publications of Schultz et. al.
[<a href="http://journals.plos.org/ploscompbiol/article/authors?id=10.1371%2Fjournal.pcbi.1004808">1</a>, <a href=http://dx.doi.org/10.1142/9789813207813_0045>2</a>].</p><p>CORDA, short for Cost Optimization Reaction Dependency Assessment is a
method for the reconstruction of metabolic networks from a given
reference model (a database of all known reactions) and a confidence
mapping for reactions. It allows you to reconstruct metabolic models for
tissues, patients or specific experimental conditions from a set of
transcription or proteome measurements.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://github.com/resendislab/corda><i class=material-icons>code</i>&nbsp;Source</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://resendislab.github.io/corda><i class=material-icons>link</i>&nbsp;Documentation</a></div></div></div><div class="mdl-cell mdl-cell--12-col-desktop mdl-cell--8-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><img src=https://github.com/cdiener/dycone/raw/master/stuff/logo.png style=height:4em;width:auto><h2 class=mdl-color-text--teal-500>Christian Diener and Osbaldo Resendis Antonio</h2><br><p>Dycone (&ldquo;dynamic cone&rdquo;) allows you infer enzymatic regulation from metabolome
measurements. It employs formalisms based on flux and k-cone analysis to connect
metabolome data to distinct regulations of enzyme activity. Most of the
analysis methods can be applied to genome-scale data.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://github.com/cdiener/dycone><i class=material-icons>code</i>&nbsp;Source</a></div></div></div></div></main></div><script src=https://storage.googleapis.com/code.getmdl.io/1.3.0/material.min.js></script></body></html>